metis-normalization/src/main/java/eu/europeana/normalization/dates/edtf/EdtfValidator.java

```diff
@@ -72,6 +73,7 @@ private static boolean validateInterval(IntervalEdtfDate intervalEdtfDate) {
   }
 
   private static boolean validateSpecificIntervalDates(EdtfDatePart startDatePart, EdtfDatePart endDatePart) {
+
     // TODO: 20/07/2022 Should we be using the java.time classes Year, YearMonth, LocalDate etc?
     // TODO: 20/07/2022 No check for null year but we check for null month and day?
```

Comment:
For more accurate checks we should check a null check for the year as well.
That means that we probably need to throw an exception as well.